contracts/index_executor/src/lib.rs

```diff
@@ -710,15 +708,14 @@ mod index_executor {
     mod tests {
         use super::*;
         // use dotenv::dotenv;
-        use ink_lang as ink;
         use phala_pallet_common::WrapSlice;
         // use pink_extension::PinkEnvironment;
         use xcm::latest::{prelude::*, MultiLocation};
 
-        fn deploy_executor() -> ExecutorRef {
+        fn deploy_executor() -> Executor {
             // Register contracts
-            let hash = ink_env::Hash::try_from([20u8; 32]).unwrap();
-            ink_env::test::register_contract::<Executor>(hash.as_ref());
+            // let hash = ink_env::Hash::try_from([20u8; 32]).unwrap();
+            // ink_env::test::register_contract::<Executor>(hash.as_ref());
```

Comment:
old code should just be deleted, not commented

Reply:
Donee